### PR TITLE
Fix crash in cnc64gdi01

### PR DESCRIPTION
**Pull request by clemty**
apparently helicopters can't .Hunt()
I introduced the bug late in testing, so it didn't trigger in review.
On the plus side, AI will now attack you from a random direction

### CODE REVIEW

**Comment by obrakmann** [comment]
Oh, I shot them down before they started the hunting, that's why I didn't see the crash. Anyway, there's no reason at all why they shouldn't be able to hunt.

**Comment by clemty** [comment]
the crash wouldn't be triggered if they come across an enemy and start attacking him on the way ther - it only triggered for me since I build the base to the right near the tiberium fields, and the helis reached their waypoint before starting .Hunt()

**Comment by obrakmann** [comment]
Looks fine :+1: . I just noticed that the biolab still has the owner row in the tooltip. You might want to fix that as well, while you're at it.

**Comment by penev92** [comment]
@obrakmann is right
>there's no reason at all why they shouldn't be able to hunt.

I can't see any either. We should probably get to the bottom of that.

File: mods/cnc/maps/cnc64gdi01/cnc64gdi01.lua [hunk 1]
```diff
@@ -176,7 +176,16 @@ end
 
 HeliHunt = function()
 	local helicopters = enemy.GetActorsByType("heli")
-	SquadHunt(helicopters)
+	local patrolpath = Utils.Random({
```
Comment (abcdefg30) [inline mods/cnc/maps/cnc64gdi01/cnc64gdi01.lua]: Can we get that extracted to a global variable at the beginning of the script? Like
```
HeliPatrolPaths =
{
	{ HeliPatrol1.Location, HeliPatrol2.Location, HeliPatrol3.Location, HeliPatrol4.Location, HeliPatrol5.Location, HeliPatrol6.Location },
	{ HeliPatrol5.Location, HeliPatrol4.Location, HeliPatrol3.Location, HeliPatrol2.Location, HeliPatrol1.Location, HeliPatrol6.Location } 
}
```
Then just use `local patrolpath = Utils.Random(HeliPatrolPaths)` here.

**Comment by clemty** [comment]
updated

**Comment by obrakmann** [comment]
> I can't see any either. We should probably get to the bottom of that.

@penev92: see #9489. Basically because our activities suck. Traits and activities regularly need to call `self.CancelActivitiy()` and stomp all over the activity queue to make themselves work.

@clemty: I swear I saw the owner row on the biolab, but maybe I'm going crazy. Should have taken a screenshot to assure myself of my sanity.

Ah, it was the husk: 
screenshot from 2015-10-01 21 17 52

**Comment by clemty** [comment]
updated

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/d8f62211d9bef94032d212a17ff5fd5781b1fb39)

**Pull request merged**